### PR TITLE
Fixing zero callout value not visible bug for cartesian charts

### PR DESCRIPTION
**Pull request by srmukher**
Work item https://uifabric.visualstudio.com/iss/_workitems/edit/9457
If Y values are zero, the callout is not displaying the values for Cartesian Charts.

Before fix:

Area Chart
image

Line Chart
image

After fix:

Area Chart
image

Line Chart
image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/d936b79dd82bffe98192bee64b5e0db28905e7df'>d936b79dd82bffe98192bee64b5e0db28905e7df</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx [hunk 1]
```diff
@@ -755,7 +755,7 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
               <div className={_classNames.calloutlegendText}> {xValue.legend}</div>
               <div className={_classNames.calloutContentY}>
                 {convertToLocaleString(
-                  xValue.yAxisCalloutData ? xValue.yAxisCalloutData : xValue.y || xValue.data,
+                  xValue.yAxisCalloutData ? xValue.yAxisCalloutData : xValue.y !== undefined ? xValue.y : xValue.data,
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx:758]: xValue.y ?? xValue.data

Comment (AtishayMsft) [reply packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx:758]: This is perfect case for a unit test

Comment (srmukher) [reply packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx:758]: yes, will keep a note

Comment (srmukher) [reply packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx]: shortened the check

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**